### PR TITLE
Review/101

### PR DESCRIPTION
**Pull request by HinsonSIDAN**
- [x] line by line review
- [ ] follow through instructions to test as a student

Have to confirm how much we can replicate at the moment before going to second review item

### CODE REVIEW

File: src/andamio/course/101/1012.md [hunk 1]
```diff
@@ -49,28 +54,25 @@ export default function MyWalletComponent() {
 ```
 
 ## You Will Know You Are Successful If:
+
 1. You can initialize a project by starting from scratch using the process outlined in Lesson 101.1, or you can use a Mesh Starter Template.
 2. You can use the `<CardanoWallet />` component.
```
Comment (HinsonSIDAN) [inline src/andamio/course/101/1012.md]: @workshop-maybe When i read through this lesson, this line makes me feel a bit disconnected. Above session just covered the `useWallet` hook, while we talk back to `<CardanoWallet />` here.

On possible approach maybe directly using something like `connect('etrnal')` here to demo connect wallet with hook.

File: src/andamio/course/101/1013.md [hunk 2]
```diff
@@ -38,37 +40,43 @@ export default function MyWalletAssetComponent() {
...
 And add a new import statement below it:
+
 ```typescript
 import MyWalletAssetComponent from "../components/MyWalletAssetComponent";
 ```
 
-Use `npm run dev` to start your development server. In a web browser, open [localhost:3000](http://localhost:3000). You should now see two copies of the same component on the page:
+Use `npm run dev` to start your development server. In a web browser, open `http://localhost:3000`. You should now see two copies of the same component on the page:
 
 ![](/course/images/101/two-components.png)
```
Comment (HinsonSIDAN) [inline src/andamio/course/101/1013.md:74]: Same question to 100's review, do we miss something here or we would like it to a production link later?

File: src/andamio/course/101/1012.md (hunk 1 above)
Comment (workshop-maybe) [reply src/andamio/course/101/1012.md]: Thanks - changing this line

File: src/andamio/course/101/1013.md (hunk 2 above)
Comment (workshop-maybe) [reply src/andamio/course/101/1013.md:74]: Localhost is good here - we're asking students to try the student repo

**Comment by workshop-maybe** [comment]
Changes look good, thanks!

File: src/andamio/course/101/1013.md (hunk 2 above)
Comment (HinsonSIDAN) [reply src/andamio/course/101/1013.md:74]: I mean the line 74, a bit confusing the comment sorry.

> ``

**Pull request merged**